src/main/webapp/app/shared/components/help-icon.component.ts

```diff
@@ -3,7 +3,7 @@ import { faQuestionCircle } from '@fortawesome/free-solid-svg-icons';
 
 @Component({
     selector: 'jhi-help-icon',
-    template: ` <fa-icon [icon]="faQuestionCircle" class="text-secondary" [placement]="placement" ngbTooltip="{{ text | artemisTranslate }}"></fa-icon> `,
+    template: ` <fa-icon [icon]="faQuestionCircle" class="text-secondary" [placement]="placement" container="body" ngbTooltip="{{ text | artemisTranslate }}"></fa-icon> `,
```

Comment:
Could adding `container="body"` here break other tooltips? I suggest you add a step to the Testing steps indicating that we should look around the application for other tooltips and check if they are still working correctly.

Reply:
`container="body"` sets the level, where the tooltip gets displayed on, to "body" (the top most level). The default behaviour is, that it is displayed on the level of the element, where it is attached to, but that leads to the tooltip being partially hidden when the div isn't big enough and the overflow is set to hidden. I think it is the expected behaviour of a tooltip to always be in front of anything else, so it shouldn't break anything.
But I think it is a good idea to verify that by testing and I will add it to the testing steps. I just went through all tooltip occurrences, that I could find, and I didn't see any breaking changes.

Reply:
Using `container body` or changing the `z index` is best practice here. So i think it looks good :+1:

Reply:
Great, just wanted to be sure 👍